packages/next/src/server/app-render/create-error-handler.tsx

```diff
@@ -84,8 +98,10 @@ export function createErrorHandler({
       }
     }
 
-    capturedErrors.push(err)
-    // TODO-APP: look at using webcrypto instead. Requires a promise to be awaited.
-    return stringHash(err.message + err.stack + (err.digest || '')).toString()
+    if (!capturedErrors.has(digest)) {
+      capturedErrors.set(digest, err)
```

Comment:
Is there ever a time where we need to know about each discrete error that occurs? This makes it possible for us to "lose" later errors if we have already encountered the same error with a certain digest. If we don't really care about the individual instances then this is fine but I don't know all the ways we consume `capturedErrors` to know if that is true.

I do think there is something inside serverInsertedHTML that generates meta tags based on some captured errors but I don't imagine there is a one to one mapping between error instance and meta tag.

Reply:
I renamed the above value to `digestErrorsMap`, it now has less errors than all captured errors, more like the 1st error for each digest, since we're only using the 1st error to throw later.

We're using another allCapturedErrors for meta tags, that one didn't get mutated.